proposed/2022/PMUIPackageSourceMapping.md

```diff
@@ -0,0 +1,89 @@
+# PMUI 
+* Start Date: 6/25/22
+* Authors: Ella McNally
+* Issue: https://github.com/NuGet/Home/issues/11797
+
+## Summary
+
+Currently there is no support for package source mapping in the PMUI. I will add a feature that allows the user to choose if they want to map the package to the source chosen when they install. I will also add error messages that tells the user if restore has problems with the mapping. This feature is a portion of what was discussed in a previous spec. https://github.com/NuGet/Home/blob/dev/implemented/2021/PackageSourceMapping.md
+
+## Motivation
+
+Adding support for package source mapping in PMUI will allow user to make/view package source mappings with more ease. It will also help the user be more aware of where their packages are coming from and what mappings they have already configured
+
+## Explanation
+
+### Functional Explanation
+
+The user will be able to choose whether or not they want to make a mapping to the source in the package source combobox in the details pane. There will be a new checkbox below the package version dropdown which the user can check if they want to map the package they are installing to the source they've selected above. The checkbox will be grayed out and will not allow the user to make a mapping if `All` sources are selected from the dropdown. It would not make sense for a user to make a mapping from one package to all sources. The added checkbox will make the details pane look like:
```

Comment:
Are we going to add a 'pin' icon to mapped, installed packages on "Installed" tab?